elements/lisk-p2p/test/integration/custom_peer_selection.ts

```diff
@@ -84,17 +84,13 @@ describe('Custom peer selection', () => {
 	beforeEach(async () => {
 		p2pNodeList = [...new Array(NETWORK_PEER_COUNT).keys()].map(index => {
 			// Each node will have the previous node in the sequence as a seed peer except the first node.
-			const seedPeers =
-				index === 0
-					? []
-					: [
-							{
-								ipAddress: '127.0.0.1',
-								wsPort: NETWORK_START_PORT + ((index + 1) % NETWORK_PEER_COUNT),
-							},
-					  ];
-
 			const nodePort = NETWORK_START_PORT + index;
+			const seedPeers = [...new Array(NETWORK_PEER_COUNT / 2).keys()]
+				.map(index => ({
+					ipAddress: '127.0.0.1',
+					wsPort: NETWORK_START_PORT + ((index + 2) % NETWORK_PEER_COUNT),
```

Comment:
Why not make all nodes seed peer of each other? That way we can completely factor out the discovery aspect from this test since it's not related.

The `peerSelectionForConnection` feature would be affected by the discovery but it doesn't appear to be tested in this file so we should try to make the discovery happen as fast as possible with more seed peers.

Reply:
same reason as the comment https://github.com/LiskHQ/lisk-sdk/pull/4240#discussion_r322816575

Reply:
> The peerSelectionForConnection feature would be affected by the discovery but it doesn't appear to be tested in this file so we should try to make the discovery happen as fast as possible with more seed peers

Before in this test, only one seed peer for there with one node. At least now it's half of the nodes as seed.